packages/fxa-auth-server/lib/profile/client.js

```diff
@@ -4,73 +4,76 @@
 
 'use strict';
 
-const isA = require('joi');
-const createBackendServiceAPI = require('../backendService');
+import axios from 'axios';
 
 const PATH_PREFIX = '/v1';
 
-// Very generic validator, because there's not really a useful response
-// here other than that it didn't fail in error
-const DeleteCacheResponse = isA.any();
+// Why is this subhub service vs "profile"?
+const serviceName = 'subhub';
 
-const UpdateDisplayNameResponse = isA.any();
+class Profile {
```

Comment:
I could have pulled this into fxa-shared but opted not too to keep things simpler.